portal/static/js/main.js

```diff
@@ -145,52 +146,52 @@ var fillViews = {
     },
     "name": function() {
         if (!$("#firstNameGroup").hasClass("has-error") && !$("#lastNameGroup").hasClass("has-error")) {
-            var content = $("#firstname").val() + " " + $("#lastname").val();
+            var content = i18next.t($("#firstname").val()) + " " + i18next.t($("#lastname").val());
```

Comment:
If this the the current users first and last name you won't want to pass it to be translated